core/src/test/scala/org/apache/spark/HeartbeatReceiverSuite.scala

```diff
@@ -153,7 +154,6 @@ class HeartbeatReceiverSuite
     heartbeatReceiverClock.advance(executorTimeout)
     heartbeatReceiverRef.askSync[Boolean](ExpireDeadHosts)
     // Only the second executor should be expired as a dead host
-    verify(scheduler).executorLost(meq(executorId2), any())
```

Comment:
Remove this because we no longer invoke `scheduler.executorLost` in `HeartbeatReceiver`'s main thread. And asserts below have already proved that `executorId2` has been expired. So, I think it should be find to remove.